daemon/cmd/daemon_main.go

```diff
@@ -667,6 +668,9 @@ func initializeFlags() {
 	flags.Bool(option.InstallEgressGatewayRoutes, false, "Install egress gateway IP rules and routes in order to properly steer egress gateway traffic to the correct ENI interface")
 	option.BindEnv(Vp, option.InstallEgressGatewayRoutes)
 
+	flags.Int(option.EgressGatewayPolicyMapEntriesName, egressmap.MaxPolicyEntries, "Maximum number of entries egress gateway's policy map")
```

Comment:
```suggestion
	flags.Int(option.EgressGatewayPolicyMapEntriesName, egressmap.MaxPolicyEntries, "Maximum number of entries in egress gateway policy map")
```

(which is what's making the travis tests fail)

Reply:
Thanks!  Sorry that I'm not a native English speaker. Updated.